misk-redis/src/main/kotlin/misk/redis/RealRedis.kt

```diff
@@ -340,8 +387,12 @@ class RealRedis(
     private val clientMetrics: RedisClientMetrics,
   ) : InvocationHandler {
     override fun invoke(proxy: Any, method: Method, args: Array<out Any?>?): Any? =
-      clientMetrics.timed(method.name) {
-        method.invoke(jedisCommand, *(args ?: arrayOf()))
+      try {
+        clientMetrics.timed(method.name) {
+          method.invoke(jedisCommand, *(args ?: arrayOf()))
+        }
+      } catch (e: InvocationTargetException) {
```

Comment:
Why this was added?

Reply:
For precise error handling and easier debugging since it wraps the exception thrown by the invoked method.